types/loadable__server/index.d.ts

```diff
@@ -104,6 +108,21 @@ export class ChunkExtractor {
 	 * Get style links as an array of React `<link>` elements
 	 */
   	getStyleElements(attr?: {} | AttrFn): Array<ReactElement<{}>>;
+
+	/**
+	 * Get inline style links as a string of <link> tags (returns a promise)
+	 */
+  	getInlineStyleTags(attr?: {} | AttrFn): Array<ReactElement<{}>>;
```

Comment:
Should probably read:
``ts
getInlineStyleTags(attr?: {} | AttrFn): Promise<Array<ReactElement<{}>>>;
```

Reply:
thanks @peterblazejewicz 32210f1